test/suites/standard1/router-config.robot

```diff
@@ -0,0 +1,94 @@
+*** Settings ***
```

Comment:
What do you think about including this in the already existing router test file? https://github.com/openshift/microshift/blob/main/test/suites/standard1/router.robot